app/src/data/index.ts

```diff
@@ -1 +1,15 @@
+import { svc } from '../services';
+
 export * from './apiDefinition';
+export type TAppContext = Awaited<ReturnType<typeof loadAppContext>>;
+export async function loadAppContext() {
```

Comment:
Why do we load it via appContext, neither directly in the documents page? It will perform unnecessary api calls for another pages, for example when user open just demo page, he don't need to load api references data.

Reply:
you are right, done